api/admin.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2022, 2023 Collabora Limited
+# Author: Jeny Sadadia <jeny.sadadia@collabora.com>
+# Author: Michal Galka <michal.galka@collabora.com>
+# Author: Guillaume Tucker <guillaume.tucker@collabora.com>
+
+"""Command line utility for creating an admin user"""
+
+import asyncio
+import argparse
+import sys
+
+import getpass
+
+from .auth import Authentication
+from .db import Database
+from .models import User, UserGroup
+
+
+async def setup_admin_group(db):
+    group_obj = await db.find_one(UserGroup, name='admin')
+    if group_obj is None:
+        print("Creating admin group...")
+        group_obj = await db.create(UserGroup(name='admin'))
+    return group_obj
+
+
+async def setup_admin_user(db, username, admin_group):
+    user_obj = await db.find_one(User, username=username)
+    if user_obj:
+        print(f"User {username} already exists, aborting.")
+        print(user_obj.json())
+        return None
+    password = getpass.getpass(f"Password for user '{args.username}': ")
+    hashed_password = Authentication.get_password_hash(password)
+    print(f"Creating {username} user...")
+    return await db.create(User(
+        username=username,
+        hashed_password=hashed_password,
+        is_admin=True,
+        groups=[admin_group]
+    ))
```

Comment:
We have removed `is_admin` field.
```suggestion
    return await db.create(User(
        username=username,
        hashed_password=hashed_password,
        groups=[admin_group]
    ))
```

Reply:
Yes, that was before the changes were merged in the User schema.  I'll update it.